### ISSUE TITLE
New sessions open in endless loop

### ISSUE THREAD

**Issue by reinhardkurz** [wardhanster/blinkin-features-n-issues#1]
upon ending the call on agent side, a new session is automatically created
in most cases the next call will not be right away but the agent will come back to BlinkIn on demand, maybe even after a few hours
I suspect that the session that is still open from before will have timed out by then and very likely will create an error
I think it would be better to actively time out sessions after a few minutes and redirect to Home - also after ending a call

**Comment by wardhanster** [comment]
Currently when the user ends the call - he is taken back to dashboard screen.
please confirm if this still creates an issue.
@reinhardkurz

**Comment by reinhardkurz** [comment]
Issue has been resolved as far as we know

Harshwardhan Kumar <notifications@github.com> schrieb am Do., 25. Juli
2019, 10:05:

**Comment by jokiatgithub** [comment]
yeah close the ticket - they reported no issues for login as well
